lib/active_record/connection_adapters/odbc_adapter.rb

```diff
@@ -1368,7 +1359,7 @@ def insert_sql(sql, name = nil, pk = nil, id_value = nil, sequence_name = nil) #
           retry_count = 0
           begin
             pre_insert(sql, name, pk, id_value, sequence_name) if respond_to?("pre_insert")
-            stmt = @connection.run(sql)
+            stmt = log(sql, name) { @connection.run(sql) }
```

Comment:
Oh awesome! That's a huge improvement. The trace output (enabled by setting `trace: true` in database.yml) is really noisy.

I'm assuming this upgrade isn't backwards compatible with 3.2? It would be sweet to get this into our query API too.